Xamarin.Forms.Build.Tasks/CompiledConverters/BindablePropertyConverter.cs

```diff
@@ -41,6 +42,16 @@ public FieldReference GetBindablePropertyFieldReference(string value, ModuleDefi
 							typeName = (ttnode as ValueNode).Value as string;
 						else if (ttnode is IElementNode)
 							typeName = ((ttnode as IElementNode).CollectionItems.FirstOrDefault() as ValueNode)?.Value as string ?? ((ttnode as IElementNode).Properties [new XmlName("", "TypeName")] as ValueNode)?.Value as string;
+					} else if (parent.XmlType.Name == "VisualState") {
```

Comment:
check that the Namespace is XFUri to avoid mismatches

Reply:
Added the check.